### PR TITLE
Cisco: qos-sai: Skip losslelss voq tests if hbm is detected

### PR DESCRIPTION
**Pull request by rraghav-cisco**
For HBM, we use the default Voq, so the split-voq tests need to be skipped. This PR addresses the Lossless Voq test.
1. Introduce a new fixture to find if HBM is enabled in the DUT, using the dshell_client.
2. Use the new fixture for the LosslessQueueVoq and lossyqueueVoq tests.

### CODE REVIEW

**Comment by wangxin** [comment]
@rraghav-cisco Is this PR a backport? Does master branch have the same issue?

**Comment by rraghav-cisco** [comment]
@wangxin : No, it is a new PR in 202205 only. Master doesn't have this code at all.

File: tests/qos/qos_sai_base.py [hunk 1]
```diff
@@ -1957,3 +1957,25 @@ def get_interface_ip(self, dut_asic):
...
 
+    def get_hbm_status(self, dut_asic, asic_index):
+        content = "print(use_hbm())"
+        dut_asic.shell("echo '{}' > /tmp/hbm.py".format(content))
+        output = dut_asic.shell(
+            "docker cp hbm.py syncd{}:/;"
+            "show platform npu script -n asic{} -s hbm.py".format(
+                asic_index, asic_index))['stdout_lines'][0]
+        return output.strip()
+
```
Comment (abdosi) [inline tests/qos/qos_sai_base.py:1968]: let's use platform check instead of determine if hbm is there or not

**Review by abdosi — CHANGES_REQUESTED** [review]
as comment

**Review by abdosi — APPROVED** [review]

**Comment by rraghav-cisco** [comment]
@wangxin : Can you pls re-submit the azp run ? I am not sure the above fail is related to my changes. Thanks in advance.

**Review by abdosi — APPROVED** [review]

**Comment by abdosi** [comment]
@wangxin please help merge this.

**Comment by rraghav-cisco** [comment]
@rraghav-cisco : Add the list of tests that are being skipped, and add the testcase skips to this PR itself.

**Review by abdosi — CHANGES_REQUESTED** [review]
PR description needs update

**Review by abdosi — APPROVED** [review]

**Pull request merged**